data/Place.csv

```diff
@@ -1,433 +1,435 @@
+PL0430,Jiangyin,"31,920658","120,284935",,2018-12-11,MG,2018-12-11,MG
+PL0431,Wuyishan,"27,756647","118,035309",,2018-12-11,MG,2018-12-11,MG
+PL0432,San Fransisco,"37,774929","-122,419418",,2018-12-11,MG,2018-12-11,MG
+PL0433,Wanzai,"22,91387","120,33538",,2018-12-11,MG,2018-12-11,MG
+PL0434,Rugao,"22,74024","120,49042",,2018-12-11,MG,2018-12-11,MG
+PL0435,Tschita,"52,03861","113,50425",,2018-12-15,MG,2018-12-15,MG
+PL0436,Livorno,"43,551891","10,30784",,2018-12-17,MG,2018-12-17,MG
+PL0437,Lavaur,"43,698792","1,81501",,2018-12-17,MG,2018-12-17,MG
+PL0438,Brno,"49,195061","16,606836",,2018-12-30,MG,2018-12-30,MG
+PL0439,Lutsk,"50,747234","25,325382",,2018-12-30,MG,2018-12-30,MG
+PL0440,Gerasimovka,"52,70488","51,50281",,2018-12-30,MG,2018-12-30,MG
+PL0441,Ulm,"48,39962","9,99661",,2019-01-08,MG,2019-01-08,MG
+PL0538,Bobai,"22,2758",109972,,2018-12-15,WH,2018-12-15,WH
+PL0539,Heilongjiang,"48,494961","127,970032",,2018-12-15,WH,2018-12-15,WH
+PL0540,Laizhou,"37,179167","119,933333",,2018-12-16,WH,2018-12-16,WH
+PL0541,Yantai,"37,4644","121,4478",,2018-12-16,WH,2018-12-16,WH
+PL0542,Quanjiao,"32,044167","118,075278",,2018-12-22,WH,2018-12-22,WH
+PL0543,Xingguo,"26,333889","115,346111",,2018-12-23,WH,2018-12-23,WH
+PL0544,Xiangxiang,"27,718649","112,550229",,2018-12-24,WH,2018-12-24,WH
+PL0545,Huai’an,"33,555605","119,112818",It’s old name was Huaiyin.,2018-12-24,WH,2018-12-24,WH
+PL0546,Linhai,"28,84","121,11",,2019-01-26,WH,2019-01-26,WH
+PL0547,Haining,"30,441944","120,573056",,2019-01-26,WH,2019-01-26,WH
+PL0548,Jiner,"31,6153",107654,,2019-01-31,LH ,2019-01-31,LH
+PL0442,Zhongxiang,"31,16368","112,57924",,2019-02-11,MG,2019-02-11,MG
+PL0443,Iserlohn,"51,372971","7,69951",,2019-02-18,MG,2019-02-18,MG
+PL0444,Baku,"40,409264","49,867092",,2019-02-19,MG,2019-02-19,MG
+PL0549,Fuzhou,"26,15","119,283333",,2019-01-28,WH,2019-01-28,WH
+PL0550,Wu’an,36698,114203,,2019-02-17,WH,2019-02-17,WH
+PL0551,Xiangshan,29475,121866,,2019-02-17,WH,2019-02-17,WH
+PL0552,Cixi,"30,166667","121,233333",,2019-02-17,WH,2019-02-17,WH
+PL0553,Dengta,"41,433333","123,333333",,2019-01-28,WH,2019-01-28,WH
+PL0554,Pingxiang,27623,"113,8545",,2019-02-23,WH,2019-02-23,WH
+PL0555,Rui’an,"27,783333",120625,,2019-02-23,WH,2019-02-23,WH
+PL0556,Tongcheng,"31,0358","116,9367",,2019-02-24,WH,2019-02-24,WH
+PL0557,Taicang,31564,121174,,2019-02-24,WH,2019-02-24,WH
+PL0558,Donghai,34552,118763,,2019-02-24,WH,2019-02-24,WH
+PL0559,Wuyuan,29248,117862,,2019-02-24,WH,2019-02-24,WH
+PL0445,Gaoyou,"32,77938","119,43173",,2019-03-08,MG,2019-03-08,MG
+PL0446,Swansea,"51,621578","-3,94337",,2019-03-09,MG,2019-03-09,MG
+PL0600,Pilsen,"49,746841","13,37699",,2019-03-08,LH,2019-03-08,LH
+PL0560,Enshi,"30,291111",109475,,2019-03-24,WH,2019-03-24,WH
+PL0561,Zhaoyuan,"37,361111","120,398806",,2019-03-24,WH,2019-03-24,WH
+PL0600,Alcalá de Henares,"40,481979","-3,363542",,2019-09-12,MG,2019-09-12,MG
+PL0601,Mumbai,19.075983,72.877655,,2019-09-13,MG,2019-09-13,MG
```

Comment:
this is the correct format, unfortunately all of the above lines have been switched to a different format:  in regex the correct format is `\d+\.?\d*,\d+\.?\d*` no `"` and `.` instead of `,` as separator.

Reply:
i think the fastest way to fix this is to copy `PL0600` and `PL0601` into a clean `Place` and adjust the format of `PL0600` manually